brave-cxf/src/main/java/com/github/kristofa/brave/cxf/ClientResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.github.kristofa.brave.cxf;
+
+import org.apache.cxf.message.Message;
+
+import com.github.kristofa.brave.http.HttpResponse;
+
+/**
+ * @author Michał Podsiedzik
+ */
+public class ClientResponse extends MessageWrapper implements HttpResponse {
+    public ClientResponse(final Message message) {
```

Comment:
please don't "final" parameters as the key word is distracting especially places like here. Add it when it says something notable (ex it is used in a anonymous class)